src/node_api/model/cli.rs

```diff
@@ -33,9 +33,15 @@ pub struct RequestDockerBuild {
     pub image: String,
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct TransparentLogOutputParams {
```

Comment:
It's called transparency log.
```suggestion
pub struct TransparencyLogOutputParams {
```